### PR TITLE
feat(article): add video shortcode

### PR DESCRIPTION
**Pull request by zhixuan666**
#270 
#222

### CODE REVIEW

File: assets/scss/partials/layout/article.scss [hunk 1]
```diff
@@ -393,6 +393,14 @@
...
         }
+        & > video {
+            position: absolute;
+            width: 100%;
+            height: 100%;
+            left: 0;
+            top: 0;
+            border: 0;
+        }
     }
```
Comment (CaiJimmy) [inline assets/scss/partials/layout/article.scss:404]: Maybe we could just use `& > iframe, & > video` selector, since both share the same style.

**Comment by zhixuan666** [comment]
fix

File: assets/scss/partials/layout/article.scss (hunk 1 above)
Comment (zhixuan666) [reply assets/scss/partials/layout/article.scss]: fix

**Comment by CaiJimmy** [comment]
👍 Thanks

**Review by CaiJimmy — APPROVED** [review]

**Pull request merged**